sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/ReleaseDownloader.java

```diff
@@ -8,11 +8,31 @@
 import android.net.Uri;
 import android.support.annotation.NonNull;
 
+import com.microsoft.appcenter.distribute.ReleaseDetails;
+
 /**
  * Interface for downloading release.
  */
 public interface ReleaseDownloader {
 
+    /**
+     * Check if downloading was already started (during the current session).
+     * <p>
+     * Note: It's NOT supposed to check status after app process restart.
```

Comment:
This comment is not true for download manager, can you explain?

Reply:
It doesn't read the status asynchronously, and is used only in cases when the status is already known.